sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpError.cs

```diff
@@ -193,6 +193,13 @@ internal static class AmqpError
                 return new EventHubsException(eventHubsResource, description, EventHubsException.FailureReason.QuotaExceeded);
             }
 
+            // The AMQP Link was closed, generally this exception would be thrown for partition specific producers and would be caused by race conditions.
```

Comment:
nit:  We may want to consider just mentioning that generally the exception occurs for a race condition between an AMQP operation and request to close a client.

Reply:
We can probably also just call it `The link` since we're in an AMQP-specific class.